documentation/design/proposals/shared-infrastructure.adoc

```diff
@@ -382,9 +376,9 @@ The following would NOT be supported:
 
 The tasks of managing brokers and routers should be offloaded to standalone components as much as possible.
 
-For router deployments, the https://github.com/interconnectedcloud/qdr-operator[qdr-operator] will be used and it supports all features required by EnMasse.
+For router deployments, the builtin mesh-forming support of the router image will be used. All router configuration except basic static config is applied using AMQP management, as the router does not have a mechanism to distinguish between static and dynamic configuration.
 
-For broker deployments, the https://github.com/rh-messaging/activemq-artemis-operator[activemq-artemis-operator] will be used, (and modified to work with our requirements). The resulting changes should be submitted upstream, although short term there may exist a temporary fork in order to make progress.
+For broker deployments, a standard upstream broker image will be used (once provided, in the meantime use existing image). All broker configuration defined in MessagingInfra is applied statically to the broker.xml and makes use of the auto-reload feature of the broker config.
```

Comment:
I'm curious to understand why using static configuration is better than the approach we already take?    We will continue to need AMQP management for monitoring purposes, so we will still need this interface.

Is leaning on this feature for making changes to a live system a supported use-case from a Broker perspective?

Reply:
Question withdrawn

Reply:
@k-wall I think the setting in StandardInfra/BrokeredInfra today is also static and requires a restart. We could make it dynamic, but its also not clear to me which entities support dynamic config over management, so I _think_ static is ok. We might want to make acceptors dynamic though.